apidoc/Titanium/Media/VideoPlayer.yml

```diff
@@ -408,6 +408,17 @@ properties:
     type: Boolean
     default: true
 
+  - name: autoHide
+    summary: Indicates if player is hidden by default and shown when its ready.
+    description: |
+        On Andorid the background of the video player is black. When switching on `autoHide` it will
```

Comment:
```suggestion
        On Android the background of the video player is set to the primary background color. When switching on `autoHide` it will
```